src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -1913,3 +1954,30 @@ class OutdatablePersistentDict(OutdatableDictMixin, PersistentStoreDict):
 
 class OutdatableDict(OutdatableDictMixin, dict):
     pass
+
+
+class ServiceSpecs(object):
```

Comment:
can we move function this class into `ServiceSpec`?

Reply:
done